.github/workflows/test-go.yaml

```
@@ -57,7 +57,7 @@ jobs:
       fail-fast: false
       matrix:
         # Kubernetes versions for setup-envtest integration tests.
-        kubernetes-version: ["1.29.3", "1.30.0", "1.31.0"]
+        kubernetes-version: ["1.29.3", "1.30.0", "1.31.0", "1.32.0"]
```

@manojks1999 Please can you bump the k8s patch version, so we can merge it ?
```suggestion
        kubernetes-version: ["1.29.3", "1.30.0", "1.31.0", "1.32.3"]
```